### DISCUSSION TITLE
Undefined index: type Error

### DISCUSSION THREAD

**Discussion by seanfiverr** [rebing/graphql-laravel#988] [General]
![issue_graph](https://user-images.githubusercontent.com/125280796/221286106-e7fff7e8-d2b7-463d-a383-8a1be75262ec.png)

Suddenly I start getting above error, can any one help what is wrong here, as it is working fine in other 2 servers but broken in one Production server.

**Comment by mfn** [comment]
That's not much to work with (besides version numbers of any software installed).

If I would have to take a guess, I'd say you changed some `fields()` definition in a type but got the nesting wrong or so.

Someone would need a lot of info to help debugging this (query, query resolver, definitions of all types involved).
